test/unit/mdc-drawer/mdc-drawer.test.js

```diff
@@ -283,8 +332,19 @@ test('adapter#focusActiveNavigationItem focuses on active navigation item', () =
   document.body.appendChild(root);
   component.getDefaultFoundation().adapter_.focusActiveNavigationItem();
 
-  const activedNavigationItemEl = root.querySelector(`.${MDCListFoundation.cssClasses.LIST_ITEM_ACTIVATED_CLASS}`);
-  assert.equal(activedNavigationItemEl, document.activeElement);
+  const activatedNavigationItemEl = root.querySelector(`.${MDCListFoundation.cssClasses.LIST_ITEM_ACTIVATED_CLASS}`);
+  assert.equal(document.activeElement, activatedNavigationItemEl);
+  document.body.removeChild(root);
+});
+
+test('adapter#focusActiveNavigationItem does nothing if no active navigation item exists', () => {
+  const {component, root} = setupTest();
+  document.body.appendChild(root);
+  const activatedNavigationItemEl = root.querySelector(`.${MDCListFoundation.cssClasses.LIST_ITEM_ACTIVATED_CLASS}`);
+  activatedNavigationItemEl.classList.remove(MDCListFoundation.cssClasses.LIST_ITEM_ACTIVATED_CLASS);
+  component.getDefaultFoundation().adapter_.focusActiveNavigationItem();
+
+  assert.notEqual(document.activeElement, activatedNavigationItemEl);
```

Comment:
Would it be better to assert that `activeElement` hasn't changed before/after?

Reply:
Done